README.md

```diff
@@ -226,6 +227,30 @@ if (!userInfo) {
 }
 ```
 
+## Strategies for Obtaining Tokens
+
+**!** Routing is **optional** strategy for token handling. Instead you can use [popup](#tokengetwithpopupoptions) or [sign widget](https://github.com/okta/okta-signin-widget).
```

Comment:
Routing is optional for the callback portion of the redirect strategy.

I see an outline something like:

- Strategies for obtaining tokens
  - Redirect to Okta
    - Overview of redirect flow
    - Handling the callback
      **- without routing
      - with hash routing
      - with path routing**
  - Using signin embedded in your application
    - using signin widget
    - custom signin form
  - User Okta-hosted signin in a popup window
  - Get tokens from an existing session (without signin prompt)


We can flush out the other sections later, but I think it's helpful to put the redirect callback handling inside a context. Many users will not need to worry about redirect or callbacks, if they choose another strategy.